design-proposals/virt-operator-deploy-instancetypes.md

```diff
@@ -0,0 +1,97 @@
+# Overview
+
+The [`common-instancetypes`](https://github.com/kubevirt/common-instancetypes)
+project provides a set of useful [instance types and preferences](https://kubevirt.io/user-guide/virtual_machines/instancetypes/) for
```

Comment:
`s/useful/pre-defined/`?

Reply:
FWIW I like useful, because that is what they are.
It's a matter of fact that they are predfeined.